Doc/c-api/long.rst

```diff
@@ -494,6 +494,17 @@ distinguished from a number.  Use :c:func:`PyErr_Occurred` to disambiguate.
    .. versionadded:: 3.13
 
 
+.. c:function:: int PyLong_GetSign(PyObject *obj, int *sign)
+
+   Get the sign of the integer object *obj*: ``0``, ``-1`` or ``+1`` for zero,
+   negative or positive integer, respectively.
+
+   Return ``0`` on success, else ``-1`` with an exception set.  This function
+   always succeeds if *obj* is a :c:type:`PyLongObject` or its subtype.
```

Comment:
Your doc doesn't mention sign. I suggest (more or less a copy/paste of C API WG doc):

```suggestion
   * On success, set *\*sign* to the integer sign, and return 0.
   * On failure, set an exception, and return -1.

   This function always succeeds if *obj* is a :c:type:`PyLongObject`
   or its subtype.
```